kotlinpoet/src/main/java/com/squareup/kotlinpoet/FunSpec.kt

```diff
@@ -359,6 +366,15 @@ public class FunSpec private constructor(
       typeVariables += typeVariable
     }
 
+    @ExperimentalKotlinPoetApi
+    public fun contextReceivers(receiverTypes: Collection<TypeName>): Builder = apply {
```

Comment:
Should we use `Iterable<TypeName>` for consistency with other methods? It's also a less-specific type.
```suggestion
    public fun contextReceivers(receiverTypes: Iterable<TypeName>): Builder = apply {
```

Reply:
Doesn't addAll take a collection?

Reply:
There's an override that takes an `Iterable`, most methods in `FunSpec.Builder` use it.